packages/lib/services/trash/index.ts

```diff
@@ -84,10 +80,7 @@ export const getTrashFolderIcon = (type: FolderIconType): FolderIcon => {
 
 export const itemIsInTrash = (item: FolderEntity | NoteEntity) => {
 	if (!item) return false;
-	if (!isTrashableNoteOrFolder(item)) return false;
-
```

Comment:
After modifying `Folder.ts` to add `deleted_time` to the definition of the virtual conflicts folder, it should be okay to remove this and similar checks. This also allows `isTrashableNoteOrFolder.ts` to be removed.